openpaas-james/apps/memory/src/main/conf/dnsservice.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one   
+  or more contributor license agreements.  See the NOTICE file 
+  distributed with this work for additional information        
+  regarding copyright ownership.  The ASF licenses this file   
+  to you under the Apache License, Version 2.0 (the            
+  "License"); you may not use this file except in compliance   
+  with the License.  You may obtain a copy of the License at   
+                                                               
+    http://www.apache.org/licenses/LICENSE-2.0                 
+                                                               
+  Unless required by applicable law or agreed to in writing,   
+  software distributed under the License is distributed on an  
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       
+  KIND, either express or implied.  See the License for the    
+  specific language governing permissions and limitations      
+  under the License.                                           
+ -->
+<!-- Read https://james.apache.org/server/config-dnsservice.html for further details -->
```

Comment:
Remove this wrong license

Reply:
As you sure we are allowed to change this license? Do you want I ask to legal?

Reply:
@rouazana Please ask if you can. I did the change but I did ask myself the same thing actually, as those files are copied from apache/james-project

Reply:
As I saw your mail exchanges with legal @rouazana , we should not change the license right?

Reply:
exactly

Reply:
Well a configuration file is required for distribution, I am unsure licenses does apply as it is a requirement to run the software.